File: csharp/sprint5/B/Solution.cs
using System;

/*
// закомментируйте перед отправкой
public class Node
{
    public int Value { get; set; }
    public Node Left { get; set; }
    public Node Right { get; set; }

    public Node(int value)
    {
        Value = value;
        Left = null;
        Right = null;
    }
}
*/

public class Solution
{
    public static bool Solve(Node root)
    {
        // Your code
        // “ヽ(´▽｀)ノ”
    }

    private static void Test()
    {
        var node1 = new Node(1);
        var node2 = new Node(-5);
        var node3 = new Node(3)
        {
            Left = node1,
            Right = node2
        };
        var node4 = new Node(10);
        var node5 = new Node(2)
        {
            Left = node3,
            Right = node4
        };
        Console.WriteLine(Solve(node5));
    }
}

File: swift/sprint5/B/solution.swift
/*
 Comment it before submitting

 open class Node {
     
     public var value: Int
     public var left: Node?
     public var right: Node?
     
     public init(_ value: Int) {
         self.value = value
         self.right = nil
         self.left = nil
     }
 }
 */

func solution(node: Node?) -> Bool {
    // Your code
    // ヽ(´▽`)/
    return false
}

func test() {
    var node1 = Node(1)
    var node2 = Node(-5)
    var node3 = Node(3)
    node3.left = node1
    node3.right = node2
    var node4 = Node(10)
    var node5 = Node(2)
    node5.left = node3
    node5.right = node4
    assert(solution(node: node5))
}
